conf/common-gradle.properties

```diff
@@ -0,0 +1,17 @@
+# suppress inspection "UnusedProperty" for whole file
+# Gradle properties are not passed to included builds https://github.com/gradle/gradle/issues/2534
+# See build-logic/gradle for plugin that does common properties distribution
```

Comment:
It's hard to dig into implementation details when I want to understand how to use it.
It'd be good to describe use cases in one place and refer to them.
Probable places: documentation, this properties file.

Reply:
added 'how to use' section to this file,
it will be better to add plugin docs, but after i extract it to separate plugin, which will be in another PR